src/typescript-service.ts

```diff
@@ -32,11 +34,30 @@ export default class TypeScriptService {
         this.root = root;
         this.projectManager = new pm.ProjectManager(root, strict, connection);
 
+        this.fetchDefinitelyTypedRepo();
+
         //initialize providers 
         this.exportedSymbolProvider = new ExportedSymbolsProvider(this);
         this.externalRefsProvider = new ExternalRefsProvider(this);
     }
 
+    /**
+     * Fetch repository from https://github.com/DefinitelyTyped/DefinitelyTyped into /tmp/DefinitelyTyped/
+     */
+    fetchDefinitelyTypedRepo() {
+        fs.exists('/tmp/DefinitelyTyped', (exists) => {
```

Comment:
If code will be moved from js/ts server itself, to its installation script, re-cloning can be scheduled